internal/orchestrator/chain_tracker.go

```diff
@@ -0,0 +1,53 @@
+package orchestrator
+
+import (
+	"context"
+	"time"
+
+	"github.com/rs/zerolog/log"
+	"github.com/thirdweb-dev/indexer/internal/common"
+	"github.com/thirdweb-dev/indexer/internal/metrics"
+)
+
+const DEFAULT_CHAIN_TRACKER_POLL_INTERVAL = 300000 // 5 minutes
```

Comment:
Don't want to make this configurable? 
This will show stale data for basically every chain

Reply:
Though about it but didn't see a reason to add another config for this. we look at metric at a minimum interval of 5min always. and since it's gauge it will remember the last value